base/healthcheck-docker/.dockerignore

```diff
@@ -0,0 +1,2 @@
+.gen
+Makefile
```

Comment:
why do we ignore the makefile?

Reply:
because it's not needed for the docker build so no reason to send it to the docker daemon

Reply:
hmm ok